frontend/packages/console-app/src/components/vpa/VerticalPodAutoscalerRecommendations.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const VerticalPodAutoscalerRecommendations: React.FC<VerticalPodAutoscalerRecommendationsProps> = ({
+  obj,
+}) => {
+  const { t } = useTranslation();
+  const [vpas] = useK8sWatchResource<any[]>({
```

Comment:
I think it would be better to create a `type` that extend `K8sResourceCommon` type for `useK8sWatchResource`  instead of using `any` which turns off type-checking.

Reply:
Updated with `K8sResourceCommon`